src/System.Net.Security/src/System/Net/Security/SslClientAuthenticationOptions.cs

```diff
@@ -14,11 +14,16 @@ public class SslClientAuthenticationOptions
         private EncryptionPolicy _encryptionPolicy = EncryptionPolicy.RequireEncryption;
         private X509RevocationMode _checkCertificateRevocation = X509RevocationMode.NoCheck;
         private SslProtocols _enabledSslProtocols = SecurityProtocol.SystemDefaultSecurityProtocols;
+        private bool _allowRenegotiation = true;
 
         internal RemoteCertValidationCallback _certValidationDelegate;
         internal LocalCertSelectionCallback _certSelectionDelegate;
 
-        public bool AllowRenegotiation { get; set; }
+        public bool AllowRenegotiation
+        {
+            get => _allowRenegotiation;
+            set => _allowRenegotiation = value;
```

Comment:
I don't understand this syntax.  Shouldn't it be:

```c#
set => _allowRenegotiation = true;
```

in order to set the default to 'true'?

Reply:
Umm, we want to set the property to the value user provides, with it being

```c#
set => _allowRenegotiation = true;
```

if someone does,

```c#
AllowRenegotiation = false;
```

the property is not going to be false, but always true.

Reply:
Ok.  Sounds good.